profile_bluesky/startup/instrument/devices/ad_imm_plugins.py

```diff
@@ -23,16 +23,16 @@
 import struct
 
 
-class IMMnLocal(Device):
-    """
-    local interface to the IMM1, & IMM2 plugins
-    """
+# class IMMnLocal(Device):
```

Comment:
remove this commented code

Reply:
Done

Reply:
instead, leave it commented since it is going away with this PR